app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class OrdersController < ApplicationController
+  before_action :authenticate_employee!
+  before_action :reward_find, only: [:create]
+
+  def create
+    if current_employee.points < @reward.price
+      redirect_to rewards_path, notice: 'You have insufficient funds'
+    else
+      @order = Order.new(employee: current_employee, reward: @reward)
```

Comment:
Let's rename helper method, and use it here instead of using callback

```suggestion
      @order = Order.new(employee: current_employee, reward: reward)
```

Reply:
good idea, it is easier to read :)